spec/menu_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'menu'
+
+describe Menu do
+  subject(:menu) { described_class.new }
+  #menu is the name of the class and includes all menu methods etc.
+  #list is the name of the actual menu within the class
+
+  describe '#view' do
+    it 'outputs the list of dishes with prices to the screen' do
+      expect{ menu.view }.to output.to_stdout
+    end
+  end
+end
+
+
+=begin
```

Comment:
Style/BlockComments: Do not use block comments. (https://github.com/bbatsov/ruby-style-guide#no-block-comments)